install.sh

```diff
@@ -246,7 +246,7 @@ supervisor_conf() {
     fi
 }
 
-if ! $packaging && [ ! -d /run/systemd/system/ ] && ! $supervisor; then
+if ! $packaging && [ ! -d /run/systemd/system/ ] && ! $supervisor && ! $without_systemd; then
```

Comment:
Isn't this double negation, evaluating to true when `without_systemd` is false and therefore it is the opposite of what we want?